libs/expo/shared/static/src/lib/regex.ts

```diff
@@ -2,4 +2,5 @@ export const Regex = {
   empty: /^\s*\S.*$/,
   date: /^(0[1-9]|1[0-2])\/(0[1-9]|[12]\d|3[01])\/(19|20)\d{2}$/,
   time: /^(?:[01]\d|2[0-3]):[0-5]\d$/,
+  email: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
```

Comment:
do we have to define this ourselves? does the form email field not offer built-in validation?

Reply:
it does not